File: app/src/main/java/li/doerf/hacked/activities/SettingsActivity.java
package li.doerf.hacked.activities;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import li.doerf.hacked.R;

/**
 * Created by moo on 01/12/15.
 */
public class SettingsActivity extends AppCompatActivity {
//    private final String LOGTAG = getClass().getSimpleName();

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);

        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        toolbar.setTitle(R.string.action_settings);
        setTitle(R.string.action_settings);

        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }

}

File: app/src/main/kotlin/li/doerf/hacked/activities/SettingsActivity.kt
package li.doerf.hacked.activities

import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import li.doerf.hacked.ui.screens.settings.SettingsScreen
import li.doerf.hacked.ui.theme.HackedTheme

class SettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        enableEdgeToEdge()
        super.onCreate(savedInstanceState)

        setContent {
            HackedTheme {
                SettingsScreen(onBack = { finish() })
            }
        }
    }
}
